rc/base/file.kak

```diff
@@ -0,0 +1,13 @@
+decl str mimetype
```

Comment:
User scripts might want to know what the current mimetype is, or even for debugging purpose (if `file` gives the current buffer an eroneous mimetype, I want a way to know what it is).